package.json

```diff
@@ -27,81 +27,81 @@
     "brace": "^0.11.1",
-    "chart.js": "^2.7.2",
+    "chart.js": "^2.7.3",
     "d3": "^5.7.0",
     "details-polyfill": "^1.1.0",
     "file-extension": "^4.0.5",
     "filesize": "^3.6.1",
-    "i18next": "^11.6.0",
-    "i18next-browser-languagedetector": "^2.2.3",
+    "i18next": "^11.10.2",
+    "i18next-browser-languagedetector": "^2.2.4",
     "i18next-icu": "^0.6.0",
     "i18next-xhr-backend": "^1.5.1",
-    "internal-nav-helper": "^1.0.2",
+    "internal-nav-helper": "^1.1.0",
     "ipfs-css": "^0.10.0",
     "ipfs-geoip": "^2.3.0",
-    "ipfs-redux-bundle": "^4.1.1",
-    "ipfs-unixfs": "^0.1.15",
-    "ipld": "^0.17.3",
+    "ipfs-redux-bundle": "^4.1.2",
+    "ipfs-unixfs": "^0.1.16",
+    "ipld": "^0.20.1",
     "ipld-explorer-components": "^1.0.0",
     "is-binary": "^0.1.0",
-    "is-ipfs": "^0.4.2",
+    "is-ipfs": "^0.4.8",
     "milliseconds": "^1.0.3",
-    "money-clip": "^2.1.0",
-    "multiaddr": "^5.0.2",
+    "money-clip": "^3.0.0",
+    "multiaddr": "^6.0.0",
     "multiaddr-to-uri": "^4.0.0",
     "prop-types": "^15.6.2",
     "pull-file-reader": "^1.0.2",
-    "react": "^16.4.2",
-    "react-ace": "^6.1.4",
+    "react": "^16.6.3",
+    "react-ace": "^6.2.0",
     "react-chartjs-2": "^2.7.4",
     "react-copy-to-clipboard": "^5.0.1",
     "react-country-flag": "^1.0.1",
-    "react-debounce-render": "^4.0.3",
-    "react-dnd": "^5.0.0",
-    "react-dnd-html5-backend": "^5.0.1",
-    "react-dom": "^16.4.2",
+    "react-debounce-render": "^5.0.0",
+    "react-dnd": "^7.0.2",
+    "react-dnd-html5-backend": "^7.0.1",
+    "react-dom": "^16.6.3",
     "react-faux-dom": "^4.2.0",
     "react-helmet": "^5.2.0",
-    "react-i18next": "^7.11.0",
+    "react-i18next": "^7.13.0",
     "react-loadable": "^5.5.0",
     "react-overlays": "^0.8.3",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "1.1.5",
-    "react-test-renderer": "^16.4.2",
-    "react-virtualized": "^9.20.1",
+    "react-scripts": "^1.0.6",
+    "react-test-renderer": "^16.6.3",
+    "react-virtualized": "^9.21.0",
     "redux-bundler": "^21.2.2",
-    "redux-bundler-react": "^1.0.1",
+    "redux-bundler-react": "^1.1.0",
     "tachyons": "^4.11.1",
     "topojson": "^3.0.2",
     "uglifyjs-webpack-plugin": "^1.3.0",
     "video-extensions": "^1.1.0",
     "window-or-global": "^1.0.1",
-    "window.ipfs-fallback": "^1.1.0"
+    "window.ipfs-fallback": "^1.2.0"
   },
   "devDependencies": {
-    "@storybook/addon-a11y": "^3.4.10",
-    "@storybook/addon-actions": "^3.4.10",
-    "@storybook/addon-knobs": "^3.4.10",
-    "@storybook/addon-links": "^3.4.10",
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "@svgr/cli": "^2.3.0",
+    "@storybook/addon-a11y": "^3.4.11",
+    "@storybook/addon-actions": "^3.4.11",
+    "@storybook/addon-knobs": "^3.4.11",
+    "@storybook/addon-links": "^3.4.11",
+    "@storybook/addons": "^3.4.11",
+    "@storybook/react": "^3.4.11",
+    "@svgr/cli": "^4.1.0",
```

Comment:
https://github.com/smooth-code/svgr/blob/master/packages/cli/CHANGELOG.md
Note to self: Need to check our generated Icon components, created via `npm run build:icons`

Reply:
Output of `build:icons` committed and pushed